eng/Set-PackageReleaseNotes.ps1

```diff
@@ -0,0 +1,17 @@
+param (
+  [Parameter(Mandatory = $true)]
+  [String]$ChangeLogLocation,
+  [String]$VersionString,
+  [String]$ReleaseNotesPropsPath
+)
+
+$ErrorActionPreference = 'Stop'
+
+try {
+  $releaseNotesForVersion = &"${PSScriptRoot}/common/Extract-ReleaseNotes.ps1" -ChangeLogLocation $ChangeLogLocation -VersionString $VersionString
```

Comment:
Do you think this wrapper script is needed? This is just capturing standard output and you could do that with the Exec task in msbuild if that is what you are going for.

Reply:
I am processing the output a bit to get rid of the .Markdown formatting. Nuget.org just takes text. so that's Why I still need this script.

Reply:
I'd suggest doing this processing to remove the markdown in the Extract-ReleaseNotes script even if it is an extra option.

Reply:
Moved all the logic into the Extract-ReleaseNotes script as you recommended returning the release notes and capturing the console output in MSBuild